src/dir.targets

```diff
@@ -174,6 +188,33 @@
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
 
+  <Target Name="CheckTestCategories"
+          AfterTargets="Test"
```

Comment:
Hmmm... OK I will try to see if there is a better way to organize these as I generally prefer a model where targets explicitly state what the depend on instead of creating this fragile target chain that is hard to plug into. For now I'm fine with what you have.